homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -307,7 +308,7 @@ def available(self):
     @property
     def unit_of_measurement(self):
         """Return the unit of measurement of this entity."""
-        return "lux"
```

Comment:
This is a breaking change; unit of measurement changed from `lux` to `lx`, which can affect things like InfluxDB.